wheel:
  index: 5
  min: -0.5
  max: 0.5
pedal:
  index: 2
  min: 0.4
  max: 0.8
max_vel: 4.16
max_yaw: 1.0
min_rate: 10
